benchmark_runner/common/template_operations/templates/hammerdb/internal_data/postgres_template.yaml

```diff
@@ -94,7 +135,7 @@ spec:
             - name: postgres-persistent-storage
```

Comment:
Is postgres-persistent-storage not needed for LSO mode?

Reply:
You right, I run LSO always with odf_pvc == True.
I have fixed it in all the database:
{%- if storage_type == 'lso' or odf_pvc == True %}